src/msw/window.cpp

```diff
@@ -1986,7 +1986,12 @@ wxWindowMSW::DoMoveSibling(WXHWND hwnd, int x, int y, int width, int height)
         }
     }
 
-    if ( !::MoveWindow((HWND)hwnd, x, y, width, height, IsShown()) )
+    if ( PositionIsOutsideShortInterval(x, y) )
```

Comment:
Again, I wonder if it wouldn't be better to have a `DoMoveWindowNoDefer()` wrapper for `::MoveWindow()` rather than repeating this test here and in `DoSetSize()` below?

Reply:
Maybe `MSWMoveWindowToAnyPosition()`? This is rather long, but describes rather well what the function does, IMHO.

I'm not sure what exactly is the problem with calling it twice, for me the `PositionIsOutsideShortInterval()` check (which might be renamed to `MSWIsDirectlySupported()` for consistency BTW) would be done inside the wrapper function (and it's very cheap, of course, so even if it's done twice, it doesn't matter).